commcare-cloud-bootstrap/bootstrap.sh

```diff
@@ -14,6 +14,6 @@ done
 commcare-cloud $ENV bootstrap-users --quiet --branch=$BRANCH
 commcare-cloud $ENV deploy-stack --skip-check --quiet -e 'CCHQ_IS_FRESH_INSTALL=1' --branch=$BRANCH
 
-## This next line is a temp fix to allow supervisor to actually get installed.
-commcare-cloud $ENV update-users --quiet --branch=$BRANCH --skip-check
```

Comment:
Do we maybe need both these lines? I think this was originally added so that the sudoers file gets updated once supervisor is installed

Reply:
Yep, added it back here: https://github.com/dimagi/commcare-cloud/pull/1752/commits/2fa3c5c3f7f4838104801fcd2c9e5f35b9bcc462